src/openzeppelin/tests/test_dual721.cairo

```diff
@@ -0,0 +1,532 @@
+use traits::Into;
+use array::ArrayTrait;
+use starknet::ContractAddress;
+use starknet::contract_address_const;
+use starknet::testing::set_caller_address;
+use starknet::testing::set_contract_address;
+use openzeppelin::token::erc721::ERC721;
+use openzeppelin::token::erc721::interface::IERC721;
+use openzeppelin::token::erc721::interface::IERC721Camel;
+use openzeppelin::token::erc721::interface::IERC721Dispatcher;
+use openzeppelin::token::erc721::interface::IERC721CamelDispatcher;
+use openzeppelin::token::erc721::interface::IERC721DispatcherTrait;
+use openzeppelin::token::erc721::interface::IERC721CamelDispatcherTrait;
+use openzeppelin::token::erc721::dual721::DualCaseERC721Trait;
+use openzeppelin::token::erc721::dual721::DualCaseERC721;
+use openzeppelin::tests::mocks::snake721_mock::SnakeERC721Mock;
+use openzeppelin::tests::mocks::camel721_mock::CamelERC721Mock;
+use openzeppelin::tests::mocks::erc721_receiver::ERC721Receiver;
+use openzeppelin::tests::mocks::erc721_receiver::SUCCESS;
+use openzeppelin::tests::mocks::erc721_receiver::FAILURE;
+use openzeppelin::tests::mocks::erc721_panic_mock::SnakeERC721PanicMock;
+use openzeppelin::tests::mocks::erc721_panic_mock::CamelERC721PanicMock;
+use openzeppelin::tests::mocks::non_implementing_mock::NonImplementingMock;
+use openzeppelin::tests::utils;
+
+use openzeppelin::tests::utils::PanicTrait;
+
+///
+/// constants
```

Comment:
What is the rule for having this lowercase and the "Selectors" comment in constants capitalized? I think the rule is to capitalize comments, having some exceptions in the function names, but this is not the case.

Reply:
Same for some other head comments in this file.